### PR TITLE
Fix compilation error during docker build

### PR DESCRIPTION
**Pull request by timemmert**
When installing fosslight_source a compiler error occurs. It is fixed by using gcc-10 instead of gcc-9 for the installation.

### CODE REVIEW

File: octrc-docker/octrc.Dockerfile [hunk 1]
```diff
@@ -180,10 +180,13 @@ RUN set -x \
 
 RUN set -x \
     # install fosslight-dependency and fosslight_binary
+    && add-apt-repository ppa:ubuntu-toolchain-r \
```
Comment (maxhbr) [inline octrc-docker/octrc.Dockerfile:183]: I dislike adding more repositories

**Comment by maxhbr** [comment]
I can not reproduce that

**Comment by maxhbr** [comment]
This issue has resolved itself?

**Comment by timemmert** [comment]
It has not occurred again.

**Pull request closed without merging**